### PR TITLE
writing readme

### PR DESCRIPTION
**Pull request by farwah1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,47 @@
...
+
+ To run this project you will need Node.js v18.10.0 and Postgres v8.7.3.
+
+## Instructions 
+
+## 1. Clone Repo
+
+Fork this repository: https://github.com/farwah1/nc-news-public and clone down to your local machine.
+
+## 2. Install Dependencies
```
Comment (Slyose) [inline README.md:22]: Just running npm i will install all dependencies listed in your package files, no need to install one-by-one!

**Review by Slyose — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -2,12 +2,12 @@
 
 ## Hosted API
 
-### To use visit: https://ncnews.cyclic.app/
+### To use visit: https://ncnews.cyclic.app
```
Comment (MKD01) [inline README.md]: This needs to have /api at the end or it will send you to an error page as you dont have an endpoint for just "/"

**Review by Slyose — APPROVED** [review]
Looks good!

**Review by MKD01 — APPROVED** [review]
Happy for this to be merged once that change has been made 👍

**Pull request merged**